`treehouses upgrade cli` returns incorrect message in Vagrant

`treehouses upgrade cli` should not return `Successfully updated cli to master branch` at the end
```
root@cli:~# treehouses upgrade cli
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t https://github.com/treehouses/cli.git
npm ERR!
npm ERR! fatal: cannot come back to cwd: Permission denied
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-05-30T02_44_12_143Z-debug.log
Successfully updated cli to master branch
```

--- a/modules/upgrade.sh
+++ b/modules/upgrade.sh
@@ -57,10 +57,9 @@ function upgrade {
         log_and_exit1 "Error: branch specified not found on cli repository"
       fi
     fi
-    npm install -g "https://github.com/treehouses/cli#${branch}"
-    echo "Successfully updated cli to $branch branch"
+    sudo npm install -g "https://github.com/treehouses/cli#${branch}" && echo "Successfully updated cli to $branch branch"
   else
-    npm install -g "@treehouses/cli@${tag}"
+    sudo npm install -g "@treehouses/cli@${tag}"
   fi
 }
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@treehouses/cli",
-  "version": "1.21.29",
+  "version": "1.21.33",
   "remote": "2346",
   "description": "Thin command-line interface for Raspberry Pi low level configuration.",
   "main": "cli.sh",

